packages/tools/devtools/devtools-view/src/components/data-visualization/EditableView.tsx

```diff
@@ -127,20 +135,26 @@ export function EditableView(props: EditableViewProps): React.ReactElement {
 			// We need to support users waiting to use "null" as a value
 			// eslint-disable-next-line unicorn/no-null
 			submitChange(null);
-			console.log("made it to null case");
 		}
 	};
 
+	/**
+	 * Return the correct type of the node. Includes special case for null since "typeof null === object"
+	 */
+	function typeOfNodeValue(nodeValue: Primitive): string {
```

Comment:
Nit: Could we move this farther up in the component so we declare it before any of our usages of it?